### PR TITLE
Update links for new site layout

### PR DESCRIPTION
**Pull request by craigbox**
[X] Docs

### CODE REVIEW

**Comment by craigbox** [comment]
Test cases:


- [x] [FAQ main page](https://deploy-preview-9765--preliminary-istio.netlify.app/latest/faq) => FAQ
- [x] [FAQ subpage](https://deploy-preview-9765--preliminary-istio.netlify.app/latest/faq/applications) => FAQ
- [x] [About/Community](https://deploy-preview-9765--preliminary-istio.netlify.app/latest/about/community) => Get Involved 
- [x] [About/Community/Join](https://deploy-preview-9765--preliminary-istio.netlify.app/latest/about/community/join) => Get Involved 
- [x] [About/Community/Customers](https://deploy-preview-9765--preliminary-istio.netlify.app/latest/about/community/customers) => Case studies
- [x] [About/Community/Partners](https://deploy-preview-9765--preliminary-istio.netlify.app/latest/about/community/partners) => FAQ 
- [ ] [Docs/Concepts/What is Istio](https://deploy-preview-9765--preliminary-istio.netlify.app/latest/docs/concepts/what-is-istio/) => Service Mesh

**Comment by craigbox** [comment]
(Ready for review/merging)

File: content/en/about/ecosystem/index.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 ---
 title: Ecosystem
-description: Ecosystem.
+description: A list of Istio providers, professional services companies and integrations.
```
Comment (shamsher31) [inline content/en/about/ecosystem/index.md:3]: nit:
```suggestion
description: A list of Istio providers, professional service companies and integrations.
```

Comment (craigbox) [reply content/en/about/ecosystem/index.md:3]: Plural is actually correct wording in this case (see Google autocomplete!); the category is "pro services" on the page too.

**Review by ericvn — APPROVED** [review]

**Pull request merged**